UA_PubSubTransportLayerMQTT symbol not available in shared library

## Description
`UA_PubSubTransportLayerMQTT` symbol not available in shared library when `BUILD_SHARED_LIBS=1` CMake option used.

## Background Information / Reproduction Steps
1. Build `open62541` as shared library with MQTT enabled
2. In your code enable MQTT:
```
UA_ServerConfig_addPubSubTransportLayer(config, UA_PubSubTransportLayerMQTT());
```
3. Link error:
```
server.cpp:(.text+0xb51): undefined reference to UA_PubSubTransportLayerMQTT
```

Used CMake options:

```bash
cmake -DBUILD_SHARED_LIBS=1 -DUA_ENABLE_PUBSUB_FILE_CONFIG=1 -DUA_ENABLE_PUBSUB_MQTT=1 ...
```

## Checklist

Please provide the following information:

 - [x] open62541 Version (release number or git tag): master branch
 - [x] Other OPC UA SDKs used (client or server): no
 - [x] Operating system: Ubuntu
 - [ ] Logs (with `UA_LOGLEVEL` set as low as necessary) attached
 - [ ] Wireshark network dump attached
 - [ ] Self-contained code example attached
 - [ ] Critical issue

--- a/plugins/ua_network_pubsub_mqtt.h
+++ b/plugins/ua_network_pubsub_mqtt.h
@@ -49,7 +49,7 @@ typedef struct {
  */    
 
 
-UA_PubSubTransportLayer
+UA_PubSubTransportLayer UA_EXPORT
 UA_PubSubTransportLayerMQTT(void);
 
 

